### PR TITLE
[wuffs] Add more json corpus

### PR DESCRIPTION
**Pull request by TheCrott**
Add more json corpus, taken from https://github.com/minio/simdjson-fuzz (contains 8024 json seeds)

### CODE REVIEW

**Comment by TheCrott** [comment]
@googlebot I signed it!

File: projects/wuffs/Dockerfile [hunk 1]
```diff
@@ -41,6 +41,13 @@ RUN mkdir rapidjson_corpus
 RUN unzip -j rapidjson.zip -d rapidjson_corpus
 RUN rm rapidjson.zip
 
+RUN git clone --depth 1 https://github.com/minio/simdjson-fuzz
+RUN mkdir simdjson_corpus
```
Comment (nigeltao) [inline projects/wuffs/Dockerfile:45]: This line and the next one could instead be:

```
RUN mv simdjson-fuzz/corpus/corpus simdjson_corpus
```

File: projects/wuffs/Dockerfile [hunk 2]
```diff
@@ -41,6 +41,13 @@ RUN mkdir rapidjson_corpus
...
+RUN cp -r simdjson-fuzz/corpus/corpus/ simdjson_corpus/
+RUN rm -rf simdjson-fuzz
+
+
```
Comment (nigeltao) [inline projects/wuffs/Dockerfile:49]: I'd collapse these 3 blank lines into one.

**Comment by nigeltao** [comment]
Looks good to me, but I don't have write access to the `google/oss-fuzz` repository.

**Pull request merged**